<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>443</width>
      <height>297</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutMain">
     <item>
      <widget class="QWidget" name="panelLeft" native="true">
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <item>
         <widget class="QLineEdit" name="lineEditIP">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutConnect">
          <item>
           <widget class="QPushButton" name="pushButtonConnect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDisconnect">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutMin">
          <item>
           <widget class="QSlider" name="sliderMin">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>11</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMin">
            <property name="text">
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditMin">
            <property name="text">
             <string>11</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutMax">
          <item>
           <widget class="QSlider" name="sliderMax">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>19</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMax">
            <property name="text">
             <string>Max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditMax">
            <property name="text">
             <string>19</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutTiming">
          <item>
           <widget class="QLabel" name="labelTiming">
            <property name="text">
             <string>Timing(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderTiming">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelTimingValue">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutStartStop">
          <item>
           <widget class="QPushButton" name="pushButtonStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelStatus">
          <property name="text">
           <string>connected</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetDados"/>
     </item>
    </layout>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>3</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
